kamelet/src/test/java/io/quarkus/ts/kamelet/KameletIT.java

```diff
@@ -0,0 +1,57 @@
+package io.quarkus.ts.kamelet;
+
+import static org.hamcrest.Matchers.is;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import io.restassured.http.ContentType;
+
+@QuarkusScenario
+class KameletIT {
+
+    @QuarkusApplication
+    static RestService app = new RestService();
```

Comment:
For these scenarios, you don't need to create the app. So, you can remove this declaration.
For rest assured, everything is configured for you using RestAssured.given()